client/ayon_maya/api/lib.py

```diff
@@ -4559,32 +4559,27 @@ def get_scene_units_settings(project_settings=None)-> tuple[str, str]:
     return linear_unit, angular_unit
 
 
-def connect_texture_reference_objects(
-        texture_connections, nodes_by_id, tx_ref_nodes_by_id):
+def apply_connections(texture_connections, nodes_by_id, target_nodes_by_id):
     """Connect texture reference object nodes to the target object
     nodes if there is one.
 
     Args:
         texture_connections (list): list of texture reference
         objects connection data
         nodes_by_id (dict): The dict with node ids.
-        tx_ref_nodes_by_id (dict): The dict with texture reference node ids.
+        target_nodes_by_id (dict): The dict with texture reference node ids.
     """
     # Compare loaded connections to scene.
     for texture_connection in texture_connections:
         source_node = next(
-            iter(tx_ref_nodes_by_id.get(
+            iter(target_nodes_by_id.get(
             texture_connection["sourceID"], [])
             ), None
         )
 
-        target_node = next(
-            iter(nodes_by_id.get(texture_connection["destinationID"], [])
-            ), None
-        )
-
+        target_nodes = nodes_by_id.get(texture_connection["destinationID"], [])
```

Comment:
There's something confusing here by the way :)....

`target_nodes_by_id` resolve to `source_node`? :) I think some variables are swapped, or some names are wrong somewhere.